components/NavBar/index.js

```diff
@@ -0,0 +1,73 @@
+import styled from "styled-components";
+import { StyledLink } from "@/components/StyledLinks";
+import { Icon } from "@/components/Icon";
+import { useRouter } from "next/router";
+
+export default function NavBar() {
+  const router = useRouter();
+  return (
+    <StyledFooter>
+      <StyledLink href="/">
+        <StyledNavIcon active={router.pathname === "/"}>
+          <Icon name="home" />
+          <StyledSubline>Home</StyledSubline>
+        </StyledNavIcon>
+      </StyledLink>
+      <StyledLink href="/createActivity">
+        <StyledNavIcon active={router.pathname === "/createActivity"}>
+          <Icon name="add" color="black" fillColor="transparent"/>
+          <StyledSubline>Add</StyledSubline>
+        </StyledNavIcon>
+      </StyledLink>
+      <StyledLink href="">
```

Comment:
Usually if wouldn't be considered good practice to build a link for a non-existent page yet, but I saw your spotlight page is in the QA now, so i would recommend you already implement this page here as if it was already existing

```suggestion
      <StyledLink href="/spotlight">
```